Mbarak-Mbigo/cp2_bucketlist
# Ch integrate travis

#### What does this PR do?
Integrate Travis-ci for automating builds check
#### Description of Task to be completed?
Integrate Travis-ci
#### How should this be manually tested?
Make a push code from one branch to develop.
Expected Results: Travis build should pass
#### Any background context you want to provide?
None
